static/js/message_edit.js

```diff
@@ -766,4 +766,22 @@ exports.handle_narrow_deactivated = function () {
     }
 };
 
+exports.move_message_to_stream = function (message_id, new_stream_id) {
+    const request = {
+        new_stream_id,
+        propagate_mode: 'change_all',
+    };
+    channel.patch({
+        url: '/json/messages/' + message_id,
+        data: request,
+        success: function () {
+
+        },
```

Comment:
The success handler should probably disable the spinner?  Probably doesn't need to do anything else.

Reply:
The spinner is not working... I'll need to fix it